### ISSUE TITLE
the  Final average score of valid navtest_v1ults is only 0.08053994634787943

### ISSUE THREAD

**Issue by raozhongyu** [swc-17/SparseDriveV2#6]
Thanks for your great work!
I try the model  on my own compute with 4 GTX5090 following your setting. However, when I tested it with navsimv1 , the  Final average score of valid navtest_v1ults is only 0.08053994634787943.   Can you provide some suggestions?

**Comment by raozhongyu** [comment]
By the way, I test the model your provide pretrained weights, the results are  same as metioned in the paper

**Comment by raozhongyu** [comment]
The reason I found the error is that the training loss became NAN during the subsequent training process.  Maybe I should change the learning rate.
